go.mod

```diff
@@ -25,8 +25,9 @@ require (
 	github.com/cosmos/cosmos-sdk v0.50.5
 	github.com/cosmos/gogogateway v1.2.0
 	github.com/cosmos/gogoproto v1.4.12
-	github.com/gagliardetto/binary v0.7.7
-	github.com/gagliardetto/solana-go v1.9.3
+	github.com/ethereum/go-ethereum v1.13.14
+	github.com/gagliardetto/binary v0.8.0
+	github.com/gagliardetto/solana-go v1.10.0
```

Comment:
@nivasan1 do we have sufficient coverage to convince us that this is ok based on CI passing?